uefi-test-runner/src/proto/shell_params.rs

```diff
@@ -0,0 +1,28 @@
+use uefi::proto::shell_params::ShellParameters;
+use uefi::table::boot::BootServices;
+use uefi::CStr16;
+
+use alloc::vec::Vec;
+
+pub fn test(bt: &BootServices) {
+    info!("Running loaded image protocol test");
+
+    let image = bt
+        .get_handle_for_protocol::<ShellParameters>()
+        .expect("No ShellParameters handles");
+    let shell_params = bt
+        .open_protocol_exclusive::<ShellParameters>(image)
+        .expect("Failed to open ShellParameters protocol");
+
+    info!("Args:");
+    for arg in shell_params.get_args_slice() {
+        let arg_str = unsafe { CStr16::from_ptr(*arg) };
+        info!("  '{}'", arg_str);
+    }
```

Comment:
nit: Since the result of getting the args is fully checked below, let's drop printing the args (many of our tests do print stuff, but it's only really necessary in cases where the output can ever change).